### PR TITLE
USWDS - Date Picker: Added aria labels to date picker components for 508 compliance. Resolves #4347

### LINKED ISSUE #4347: Code sample for date range picker results in 508 non-compliance
## Description

Noticed that both the component preview and component code section results in a 508 compliance error due to the misusage of the aria-describedby attribute and a missing aria-labeledby attribute.

The input tag should be
`<input
        class="usa-input"
        id="event-date-start"
        name="event-date-start"
        type="text"
        aria-labeledby="event-date-start-label"
        aria-describedby="event-date-start-hint"
      />`

## Steps to reproduce the issue

N/A

## Additional information [optional]
image

### PR DESCRIPTION
**Pull request by mahoneycm**
## Description

Adds `aria-labelledby` tag to date picker components to resolve form label 508 error as suggested by @bmcorum. Resolves #4347 

## Additional information

**Before:**

image

**After**

image

### CODE REVIEW

File: src/components/test/input-disabled.njk [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    <label class="usa-label" for="file-input-single">Input accepts a single file</label>
+    <input id="file-input-single" class="usa-file-input" type="file" name="file-input-single" />
+  </div>
+
+  <div class="usa-form-group">
+      <label class="usa-label" for="file-input-single">Input accepts a single file</label>
+      <input id="file-input-single" class=" usa-file-input usa-file-input--disabled usa-file-input__box" type="file" name="file-input-single" />
+  </div>
+
+</main>
```
Comment (mejiaj) [inline src/components/test/input-disabled.njk:35]: @mahoneycm was this meant to be included?

Comment (mahoneycm) [reply src/components/test/input-disabled.njk:35]: @mejiaj no my apologies! Was working on something in a different branch and thought I omitted that from my commit

Comment (mejiaj) [reply src/components/test/input-disabled.njk]: All good, thank you!

**Review by mejiaj — APPROVED** [review]
Thanks for updating this!

**Pull request merged**